bin/picca_xcf_angl.py

```diff
@@ -253,7 +253,7 @@ def main(cmdargs):
     xcf.lambda_abs = constants.ABSORBER_IGM[args.lambda_abs]
 
     # read blinding keyword
-    blinding = io.read_blinding(args.in_dir)
+    blinding = io.read_blinding(args.in_dir, args.lambda_abs, args.lambda_abs_obj)
```

Comment:
I would have expected the second argument here would be 'obj_name' as in picca_xcf, but I noticed that this variable is named different in this (not very popular) script, so we can leave it as is for now.